### PR TITLE
DUI: small fixes

### PR DESCRIPTION
**Pull request by aosyatnik**
Some little bugs, these were sent by @elayabharath .

Reviewers
@Benglin , please take a look.

### CODE REVIEW

File: src/DynamoCoreWpf/UI/Themes/Modern/SidebarGridStyleDictionary.xaml [hunk 1]
```diff
@@ -8,13 +8,8 @@
...
                      TargetType="{x:Type Thumb}">
         <Grid>
-            <Rectangle HorizontalAlignment="Stretch"
-                       VerticalAlignment="Stretch"
-                       Width="Auto"
-                       Height="Auto"
-                       Fill="Transparent" />
             <Border x:Name="Rectangle1"
-                    HorizontalAlignment="Stretch"
+                    HorizontalAlignment="Right"
```
Comment (aosyatnik) [inline src/DynamoCoreWpf/UI/Themes/Modern/SidebarGridStyleDictionary.xaml]: Remove the margin on the right of the scrollbar.
image

File: src/DynamoCoreWpf/Controls/DynamoTextBox.cs [hunk 2]
```diff
@@ -360,7 +360,7 @@ public LibraryToolTipPopup()
             this.CustomPopupPlacementCallback = PlacementCallback;
             this.DataContext = null;
             this.Child = tooltip;
-            this.dispatcherTimer.Interval = new TimeSpan(0, 0, 0, 0, 500);
+            this.dispatcherTimer.Interval = new TimeSpan(0, 0, 0, 0, 250);
```
Comment (aosyatnik) [inline src/DynamoCoreWpf/Controls/DynamoTextBox.cs]: Tooltip on hover out: Reduce the timer to half of what is present now.

File: src/DynamoCore/Properties/Resources.en-US.resx [hunk 3]
```diff
@@ -587,7 +587,7 @@ value: bool = false</value>
     <value>Nothing is selected.</value>
   </data>
   <data name="SearchTextBlockText" xml:space="preserve">
-    <value>Search Library</value>
+    <value>SEARCH LIBRARY</value>
```
Comment (aosyatnik) [inline src/DynamoCore/Properties/Resources.en-US.resx]: Capitalize the ‘Search library’ to ‘SEARCH LIBRARY’

File: src/DynamoCoreWpf/Views/Search/LibraryView.xaml [hunk 4]
```diff
@@ -708,12 +707,12 @@
                                             Property="Foreground"
                                             Value="#AAABAB" />
                                 </Trigger>
-                                <Trigger Property="Button.IsPressed"
-                                         Value="True">
+                                <DataTrigger Binding="{Binding ContextMenu.IsOpen, RelativeSource={RelativeSource Self}}"
```
Comment (aosyatnik) [inline src/DynamoCoreWpf/Views/Search/LibraryView.xaml]: Remove the background color for normal state of the Add button at the bottom.
image

File: src/DynamoCore/Utilities/Configurations.cs [hunk 5]
```diff
@@ -208,8 +208,7 @@ public class Configurations
         public const string CategoryGroupAction = "Actions";
         public const string CategoryGroupQuery = "Query";
         public const string CategoryDelimiterString = ".";
-        public const char ShortenedCategoryDelimiter = '-';
-        public const string CategoryDelimiterWithSpaces = " - ";
```
Comment (aosyatnik) [inline src/DynamoCore/Utilities/Configurations.cs]: There is an extra space between ‘-‘ and ‘Create’ under search results.
image

CategoryDelimiterWithSpaces was used to create this type of space. I removed it.

File: src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml [hunk 6]
```diff
@@ -559,8 +559,7 @@
                                                                FontSize="12"
                                                                Margin="16,8,0,8">
                                                             <Run Text="{Binding Prefix, Mode=OneWay}"
-                                                                 MouseDown="OnPrefixTextBlockMouseDown" />
-                                                            <Run Text="{Binding GroupName, Mode=OneWay}">
+                                                                 MouseDown="OnPrefixTextBlockMouseDown" /><Run Text="{Binding GroupName, Mode=OneWay}">
```
Comment (aosyatnik) [inline src/DynamoCoreWpf/Views/Search/LibrarySearchView.xaml]: @Benglin , do you remember task, when I had to change inputs for tooltip. And `Run` had some default margin, that I couldn't remove. So, I changed 2 Runs in 2 TextBlocks. Now I know, why `Run` had this space. Answer is [here](http://stackoverflow.com/questions/11090084/how-to-get-rid-of-whitespace-between-runs-in-textblock).

**Comment by Benglin** [comment]
Thanks @aosyatnik for the fixes, and also the note about spaces around `Run`, now that's clear. LGTM! :)

**Pull request merged**